### DISCUSSION TITLE
Filament Broken - Buttons always loading, forms not submitting, etc

### DISCUSSION THREAD

**Discussion by joelwmale** [filamentphp/filament#8842] [Help] [answered]
Have got a project that was previously running on Filament 2, and worked no issues at all, but now, having upgrading to v3 I get very strange errors, like not even being able to log in:

![CleanShot 2023-10-03 at 16 03 17](https://github.com/filamentphp/filament/assets/3906839/5c903112-5aa6-45af-a6ad-fd15cb5e5bf1)

This is the default state for the panel, as you can see, it has a loading indicator just on page load.

Entering any username and any password, just simply reloads the page.

Getting no console errors:

![CleanShot 2023-10-03 at 16 04 11](https://github.com/filamentphp/filament/assets/3906839/6582eea0-a7e5-43bf-b783-6712cb75c25e)

Using the default panel configuration:

```php
    public function panel(Panel $panel): Panel
    {
        return $panel
            ->default()
            ->id('admin')
            ->path('admin')
            ->login()
            ->colors([
                'primary' => Color::Amber,
            ])
            ->discoverResources(in: app_path('Filament/Resources'), for: 'App\\Filament\\Resources')
            ->discoverPages(in: app_path('Filament/Pages'), for: 'App\\Filament\\Pages')
            ->pages([
                Pages\Dashboard::class,
            ])
            ->discoverWidgets(in: app_path('Filament/Widgets'), for: 'App\\Filament\\Widgets')
            ->widgets([
                Widgets\AccountWidget::class,
                Widgets\FilamentInfoWidget::class,
            ])
            ->middleware([
                EncryptCookies::class,
                AddQueuedCookiesToResponse::class,
                StartSession::class,
                AuthenticateSession::class,
                ShareErrorsFromSession::class,
                VerifyCsrfToken::class,
                SubstituteBindings::class,
                DisableBladeIconComponents::class,
                DispatchServingFilamentEvent::class,
            ])
            ->authMiddleware([
                Authenticate::class,
            ]);
    }
```

I don't have Livewire installed (so using the Livewire version from filament) and have tried clearing cache/incognito but nothing at all is working

@awcodes did attempt to help me in Discord, but couldn't work out what was going on, so posting here in hopes of having someone who may have seen and fixed this issue before.

**Comment by leandrocfe** [comment]
Can you share this project on Github without resources/pages?

**Reply by joelwmale** [reply]
Hey, yeah try this: https://github.com/joelwmale/filament-test

**Reply by joelwmale** [reply]
Just pull it down, composer install and go to `filament-test.test/admin` and watch the magic happen

**Reply by leandrocfe** [reply, marked as answer]
livewire v3 config is different. Run `php artisan livewire:publish --config` to publish/override the config

**Reply by joelwmale** [reply]
Does that fix the issue for you?

**Reply by leandrocfe** [reply]
```php
'layout' => 'components.layouts.app',
'inject_assets' => true,
```

**Reply by leandrocfe** [reply]
run the command or change these params in the config/livewire.php file

**Reply by joelwmale** [reply]
Oh wow that is actually the issue, strangely enough! Thank you!
